ext/standard/http.c

```diff
@@ -360,3 +361,27 @@ PHP_FUNCTION(request_parse_body)
 	SG(request_parse_body_context).throw_exceptions = false;
 	memset(&SG(request_parse_body_context).options_cache, 0, sizeof(SG(request_parse_body_context).options_cache));
 }
+
+PHP_FUNCTION(http_get_last_response_headers)
+{
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
+
+	if (!Z_ISUNDEF(BG(last_http_headers))) {
+		RETURN_COPY_VALUE(&BG(last_http_headers));
```

Comment:
Should be RETURN_COPY, not RETURN_COPY_VALUE. Otherwise you don't increment the refcount. That fixes the test for me. It worked when you used ZVAL_COPY(return_value, ...) which is the equivalent of RETURN_COPY.